agent/agent.go

```diff
@@ -219,16 +220,38 @@ func (a *Agent) Run(ctx context.Context) error {
 	a.header = a.buildTransportHeaderMap()
 
 	// Fail the agent after startup if the id is invalid
+	logger.Debug("validating agent name")
```

Comment:
Do we still want to keep these debug lines in here?

Reply:
Yeah, I think they're actually useful